### DISCUSSION TITLE
How to modify the Swiss ISO?

### DISCUSSION THREAD

**Discussion by thor171** [emukidid/swiss-gc#540] [Q&A]
Hi,
I'm wondering if it's possible to modify/add some files to the iso file which is provided.
My GC has a Viper Extreme Chip, but the USB flashing is not working anymore, it stops right after some flash progress and cobra manager says the bios is not responding.
So I want to burn a DVD in order to copy the swiss file for the Viper Flash in the Cobra File Browser.

**Comment by emukidid** [comment]
I don't recall if you can flash a DOL file from a ISO9660 disc in the Cobra BIOS, but if you can, just burn a copy of the Swiss distribution extracted to the root of a DVD-R and see if there's an option to do so.

**Reply by thor171** [reply]
Indeed this is possible, at least with Cobra 2.1. I know it, because I did so some time ago, I flashed Swiss 533 from DVD, because of the USB flasher not working anymore.
Burning a disc as standard ISO9660 is just working fine, I had to test different blank disks I had at home and found one which worked like a charm. I'm now on version r1086 :)

Great job by the way, r533 was pretty buggy but 1086 is working great.
I'm using the serial port adapter and a Sandisk Ultra 128GB@32MHz. No stuttering anymore, even the cutscenes are running smooth!

Do you know, why some of the Viper Extreme compressed swiss dols don't work? For example I flashed r1026 when one of the discs I burned was read once, but when starting the dol the screen went green and nothing happened, it freezed. I remember experiencing this before, when I flashed r563.
